.gitignore

```diff
@@ -41,3 +41,4 @@ nb-configuration.xml
 
 ##Ignore JRebel project configuration
 rebel.xml
+.checkstyle
```

Comment:
@terrywbrady : this looks good/obvious. However, for consistency sake, can we move this up & apped it to the section of the `.gitignore` that is specific to Eclipse?  If you look closely, we've split up this `.gitignore` into sections per editor/IDE.  The Eclipse section is up on line 7: https://github.com/DSpace/DSpace/blob/master/.gitignore#L7